src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -29,41 +29,47 @@ export const RatingImg = styled.img`
 `
 export const CheckImg = styled.img`
   width: auto;
-  margin: 0 30px;
+  margin-top: 10px;
+  // margin: 0 30px;
 
   @media (max-width: 960px) {
     position: relative;
     height: 100px;
-    bottom: -20px;
-    width: 94px;
+    // bottom: -20px;
+    width: 80px;
+    height: auto;
   }
 
   @media (min-width: 961px) {
-    width: 20vw;
+    width: 80px;
     height: auto;
     bottom: 0;
     right: 50vw;
   }
```

Comment:
Currently this `bottom` and `right: 50vw` is not doing anything since the element position is "static". Firefox says to me that it's not being applied to the element. Then I think can be removed

![imagen](https://user-images.githubusercontent.com/66505715/158848006-63b3e766-98b6-4f71-aa9e-3fc108f0e196.png)

If you want to know, the position is static because position relative is inner of `media query max-width 960px`, then it's not applying in this size. But we already have seen that this size looks good without position relative, bottom and right position, then please considerate if you want to add position relative or else please remove bottom and right property, since they're applying to the element